docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -46,6 +60,16 @@ To discover AWS cloud resources, Octopus uses the following variables:
 | `Octopus.Aws.AssumedRole.SessionDuration` | N        | The maximum duration the session will be available for if assuming a role during discovery. If not set then the default duration from the IAM role will be used.                                                                                             |
 | `Octopus.Aws.AssumedRole.ExternalId`      | N        | An external ID to use if assuming a role during discovery. See the AWS documentation for more information on the use of external IDs.                                                                                                                        |
 
+From **Octopus 2022.3**, AWS steps that support target discovery will allow you to configure the variables above from within the step configuration if they have not been configured within your project yet:
+
+- Open the deployment process and navigate to the AWS step
+- Open the Cloud Connections section
+- Click the Amazon Web Services tile. If target discovery variables have not been configured this will say "Configure Now".
+- Enter the credentials in the drawer
+- Click Ok. This will save the well-known variables above into your project.
```

Comment:
```suggestion
1. Open the deployment process and navigate to the AWS step
1. Expand the **Cloud Connections section.
1. Click the **Amazon Web Services** tile. If target discovery variables have not been configured, this will say **Configure Now**.
1. Choose the credentials to use for target discovery.
1. Click **Ok**. This will save the well-known variables above into your project.
```